### PR TITLE
fix: add full stop after each item of content guidance

### PR DESCRIPTION
**Pull request by weronika-szalas**
## Description

Add full stop after each item of content guidance (unless it already has a full stop which is sometimes the case then only add a space to it).

Fixes [PUPIL-1067](https://www.notion.so/oaknationalacademy/Content-guidance-rendering-lesson-overview-page-10026cc4e1b18028a154da30451ca249)

## How to test

1. Navigate to a lesson with content guidance
2. Content guidance block of text with should have full stop after each line of content guidance.

Check this lessons:
- https://deploy-preview-2974--oak-web-application.netlify.app/pupils/programmes/drama-primary-year-3-l/units/stories-shared-across-the-world-2d56/lessons/dragons-c8v3cc/overview (content guidance item already has a full stop coming from database)
- https://deploy-preview-2974--oak-web-application.netlify.thenational.academy/pupils/programmes/english-secondary-year-10-aqa/units/macbeth-lady-macbeth-as-a-machiavellian-villain/lessons/an-exploration-of-act-1-scenes-1-4/overview (content guidance item does not have a full stop coming from database)

## Screenshots

How it used to look (delete if n/a):
screenshot in the ticket

How it should now look:
Screenshot 2024-11-07 at 17 18 19

### CODE REVIEW

**Comment by RBrownsett** [comment]
@weronika-szalas looking good - just wondering is there a way not to add a full stop if one already exists? For example this lesson now has two full stops after the text 'Contains sexual content' 

image

Link to lesson:
https://deploy-preview-2974--oak-web-application.netlify.thenational.academy/pupils/programmes/rshe-pshe-secondary-year-10-l/units/online-and-media-dangers-of-viewing-explicit-material-6a7c/lessons/impact-on-attitudes-and-behaviour-69h6ct/overview

**Comment by weronika-szalas** [comment]
@RBrownsett good shout, just spotted this too and pushed a fix! Should be up there once it rebuilds. I basically checked if the last character is a full stop:
- if it is add only a space
- if it isn't add full stop and a space

Do you think there can be any other scenarios?

**Comment by RBrownsett** [comment]
@weronika-szalas looks good to me - I can't think of any other scenarios but I will check in areas where there is a lot of content guidance

File: src/components/PupilViews/PupilLessonOverview/PupilLessonOverview.view.tsx [hunk 1]
```diff
@@ -113,6 +113,8 @@ export const PupilViewsLessonOverview = ({
       </OakP>
     ));
 
+  console.log(contentGuidance);
```
Comment (RBrownsett) [inline src/components/PupilViews/PupilLessonOverview/PupilLessonOverview.view.tsx:116]: @weronika-szalas just checking - do we need to keep this console.log or is it just while test the different scenarios?

Comment (kimon-satan) [reply src/components/PupilViews/PupilLessonOverview/PupilLessonOverview.view.tsx:116]: yes pls remove this

**Review by kimon-satan — APPROVED** [review]

File: src/components/PupilViews/PupilLessonOverview/PupilLessonOverview.view.tsx (hunk 1 above)
Comment (weronika-szalas) [reply src/components/PupilViews/PupilLessonOverview/PupilLessonOverview.view.tsx]: forgot to remove, done!

**Review by RBrownsett — APPROVED** [review]
Tested the different scenarios - looks good!

**Pull request merged**